### PR TITLE
 feat(manuals): add manuals of plugins and clientset

### PR DESCRIPTION
**Pull request by kdada**
**What this PR does / why we need it**:

- Rename some chapters.
- Add manuals of `version` and `healthcheck`
- Add best-practice of `ClientSet`

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @ddysher 
/cc @zjj2wry 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: manuals/zh-hans/concepts/clients.md [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+
+### 创建 ClientSet
+
+在 `clientset` 中创建 `clientset.go`：
+```
+$ cd $GOPATH/src/clientset
+$ touch clientset.go
+```
+
+`clientset.go` 代码如下：
```
Comment (zjj2wry) [inline manuals/zh-hans/concepts/clients.md]: 这个得自己写吗？

Comment (kdada) [reply manuals/zh-hans/concepts/clients.md]: Of course.

Comment (zjj2wry) [reply manuals/zh-hans/concepts/clients.md]: ok，我觉得可以先这样。但是最好还是可以自动生成，因为 k8s 的 clientset 要自己写 register.go 被吐槽无数次了。😁

File: manuals/zh-hans/SUMMARY.md [hunk 2]
```diff
@@ -18,14 +16,19 @@
...
+  * [配置器机制](concepts/configurer.md)
+  * [插件机制](concepts/plugin.md)
+  * [多客户端整合](concepts/clients.md)
 * 插件
   * [Logger](plugins/logger.md)
   * [Reqlog](plugins/reqlog.md)
   * [Metrics](plugins/metrics.md)
   * [Tracing](plugins/tracing.md)
   * [Profiling](plugins/profiling.md)
+  * [Version](plugins/version.md)
```
Comment (ddysher) [inline manuals/zh-hans/SUMMARY.md]: 不如都用中文吧？

```
版本
健康检查
```

File: manuals/zh-hans/plugins/version.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# Version 插件
```
Comment (ddysher) [inline manuals/zh-hans/plugins/version.md]: 版本插件

File: manuals/zh-hans/plugins/healthcheck.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
+# Health Check 插件
```
Comment (ddysher) [inline manuals/zh-hans/plugins/healthcheck.md]: 健康检查插件

**Comment by kdada** [comment]
@ddysher  PTAL

**Pull request merged**